### ISSUE TITLE
Additional components with dat.GUI style

### ISSUE THREAD

**Issue by edankwan** [wearekuva/oui#9]
It doesn't seem to be a way to add additional components with dat.GUI style

**Comment by marklundin** [comment]
this is dependent on #8

**Comment by marklundin** [comment]
Fixed in [v0.0.14](https://github.com/wearekuva/oui/tree/v0.0.14). Constraint objects passed via `annotate` or `datoui.add` can now contain a `control` indicating the controller that should be used instead of the default.

Here's how you'd select the `ColorPicker` datoui style
```javascript
let setting = { someColor: [1, 0, 0.5 ]}
gui.add( obj, 'propName', { control: oui.controls.ColorPicker })
```
or using plain `oui` with decorators

```javascript
oui({
  @oui.color()
  someColor: [1, 0, 0.5 ]
})
```
